app/models/commit.rb

```diff
@@ -0,0 +1,10 @@
+class Commit < ActiveRecord::Base
+  attr_accessible :commit_oid, :repository
+  attr_accessible :author, :author_date
+  attr_accessible :commit_date, :committer
+
+  has_many :ontology_versions
+  has_many :ontologies, through: :ontology_versions
+
+  belongs_to :repository
+end
```

Comment:
Really? The only thing? I could think of a few. However i did not include them, as we do not need them right now...

e.g. commit-message, files affected by commit, the diff of the commit, the way files were affected (added, deleted, changed, rewritten).